extensions/Bookalope/css/bookalope-darkest.css

```diff
@@ -1,5 +1,9 @@
 @charset "utf-8";
 
+.spectrum--dark {
+    --spectrum-fieldlabel-m-asterisk-color: inherit;
+}
```

Comment:
This is a class `.spectrum-dark` in the `bookalope-darkest.css` file — not called `.spectrum-darkest`? And there isn’t such a class in the two light/lightest css files?

Reply:
I've fixed the class name for `bookalope-darkest.css`.
No, there is no such class for light/lightest themes.